apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+    return (
+      <div style={{textAlign: 'center', verticalAlign: 'middle'}}>
+        {teachesCs ? teachesCs : "Null"}
+      </div>
+    );
+  };
+
+  formatComment = (comment) => {
+    return (
+      <div>
+        {comment}
+      </div>
+    );
+  };
+
+  mapHistoricSummary = (teaches_cs) => {
+    switch (teaches_cs) {
+      case null:
+        return 'Null';
+      case undefined:
+        return 'No Data';
+      default:
+        return teaches_cs.charAt(0).toUpperCase() + teaches_cs.slice(1).toLowerCase();
+    }
+  };
+
+  formatSchool = (school) => {
+    return (
+      <div>
+        {school.name}
+        <br />
+        ({school.city}, {school.state})
+      </div>
+    );
+  };
+
+  beginReviewButton = (_value, data) => {
+    if (this.state.resolvedReports.includes(data.rowData.id)) {
+      return (
+        <div style={{textAlign: 'center', verticalAlign: 'middle'}}>
+          Review Completed
+        </div>
+      );
+    } else {
+      return (
+        <Button
+          onClick={() => {this.setState({toReview: data.rowData}); $('#report-header')[0].scrollIntoView();}}
+          size="large"
+          text="Review this School"
+        />
+      );
+    }
+  };
+
+  showMainList = () => {
+    this.setState({
+      toReview: undefined,
+      notes: ''
+    });
+    $('#report-header')[0].scrollIntoView();
+  };
+
+  render() {
+    let display;
+    if (this.state.toReview) {
+      display = this.renderReview();
+    } else {
+      display = this.renderTable();
+    }
+
+    return (
+      <div>
+        <h1 id="report-header">
+          Reported Access Report Inaccuracies
+        </h1>
+        {display}
+      </div>
+    );
+  }
+
+  stringForExplanationLabel = {
+    offers_ap: "Offers AP CS?",
+    offers_ib: "Offers IB CS?",
+    consistent_teacher_surveys: "Consistent Teacher/Admin Surveys",
+    state_offering: "Teaches per State Data?",
+    consistent_non_teacher_surveys: "Consistent Non-Teacher/Admin Surveys",
+    inconsistent_surveys: "Inconsistent Surveys",
+    last_years_summary: "Last Year's Summary",
+    two_years_agos_summary: "Two Years Ago's Summary",
+  };
+
+  renderReview = () => {
+    const noData = 'No Data';
+    const school = this.state.toReview.school;
+    const auditData = this.state.toReview.summary_audit;
+    const explanation = auditData.explanation;
+    const highlightedStyle = {
+      backgroundColor: "#8afc9b",
+    };
+
+    const overrideText = `Create a ${this.state.toReview.teaches_cs ? 'YES' : 'NO'} override for this school.`;
+    return (
+      <div style={{maxWidth: 615}}>
+        <h2>
+          {school.name} ({school.city}, {school.state})
+        </h2>
+        <Button
+          onClick={this.showMainList}
+          size="large"
+          text="Back to main list"
+        />
+        <br />
+        <hr />
+        <div>
+          <h3>We currently classify this school as <b>{auditData.previous_years_results[0] || 'NULL'}</b> based on:</h3>
```

Comment:
We use variations of the `'NULL'` string in several places, all with inconsistent capitalization (Null, NULL). Can we standardize?

Reply:
This is the only place where we use NULL. The current classification of the school is in all caps (YES, NO, etc.) to make it stand out. Using NULL does the same when the value is `null`.

Reply:
Ok, like above `null` in any form seems rather technical. Depending on the consumer of this UI, it might be better to provide something more descriptive. Your call.